### PR TITLE
Support SVG image, other related cleanup

### PR DESCRIPTION
**Pull request by zpao**
I got tired of dealing with PRs trying to add image. Then I decided to cleanup while I was there since we've had to do a few followup changes when changes in getMarkupWrap weren't made in both places.

### CODE REVIEW

File: src/isomorphic/classic/element/ReactDOM.js [hunk 1]
```diff
@@ -157,6 +157,7 @@ var ReactDOM = mapObject({
   defs: 'defs',
   ellipse: 'ellipse',
   g: 'g',
+  iamge: 'image',
```
Comment (cody) [inline src/isomorphic/classic/element/ReactDOM.js]: `iamge` is a typo.

Comment (zpao) [inline src/isomorphic/classic/element/ReactDOM.js]: Bah, I caught it and fixed it but forgot to save the file :(

**Comment by spicyj** [comment]
Sure.

**Comment by lhausermann** [comment]
Great. Thanks

**Comment by abhilashsajeev** [comment]
When will be this available in npm?

**Comment by jimfb** [comment]
@abhilashsajeev This is slated for the 0.14 beta release, which is coming very soon.  If you're anxious to try it out, you can always check out the source code and do a grunt build.

**Comment by abhilashsajeev** [comment]
@jimfb  Okey Jim. What is the expected date of release for 0.14 ? I need to use it in production.

**Comment by spicyj** [comment]
@abhilashsajeev If you need it urgently, you can always patch your local copy of React to include this change. 0.14 final will probably be released sometime in July.

**Comment by abhilashsajeev** [comment]
@spicyj  July first week is fine. If I patch my existing version to production will there be any problem like breaking of stable parts etc?

**Comment by spicyj** [comment]
No, not July first week. Sometime in July.

**Comment by abhilashsajeev** [comment]
Oh.. Then I should probably go with patch. There was some problem with using d3 also with react.

**Pull request merged**